src/azure-cli/azure/cli/command_modules/network/_validators.py

```diff
@@ -1911,3 +1911,23 @@ def process_private_endpoint_connection_id_argument(cmd, namespace):
     namespace.resource_provider = '{}/{}'.format(result['namespace'], result['type'])
     namespace.name = result['child_name_1']
     del namespace.connection_id
+
+
+def process_vnet_name_or_id(cmd, namespace):
```

Comment:
duplicated work?
https://github.com/Azure/azure-cli/blob/e3b87446b52e8b85e55a8812ec56ae1f97787837/src/azure-cli/azure/cli/command_modules/network/_validators.py#L86-L111

Reply:
I cannot use it since it's a list and SubResource. `ids.append(SubResource(id=val)) `